### ISSUE TITLE
[bitnami/nginx-ingress-controller] MaxmindDB download fails because of read-only dir.

### ISSUE THREAD

**Issue by balazs92117** [bitnami/charts#21302]
### Name and Version

bitnami/nginx-ingress-controller 9.9.3

### What architecture are you using?

amd64

### What steps will reproduce the bug?

If I set the `maxmindLicenseKey`, the download of MaxmindDB fails because of the latest modification of the chart: #21196
I know this is just a workaround, but as it maps the emptyDir as `readOnly: true`, the download fails, because the files can't be written into the folder.

### Are you using any custom parameters or values?

Only the `maxmindLicenseKey: XXX` value.

### What is the expected behavior?

The download of Maxmind DBs should work.

### What do you see instead?
```
I1129 10:49:39.403690       1 flags.go:387] "downloading maxmind GeoIP2 databases"
E1129 10:49:40.554307       1 maxmind.go:74] GeoLite2-City.mmdb not found
W1129 10:49:40.554321       1 store.go:1214] The GeoIP2 feature is enabled but the databases are missing. Disabling
```

**Comment by carrodher** [comment]
Thank you for opening this issue and submitting the associated Pull Request. Our team will review and provide feedback. Once the PR is merged, the issue will automatically close.

Your contribution is greatly appreciated!

**Comment by balazs92117** [comment]
Any update?

**Comment by joancafom** [comment]
Hi @balazs92117 

The associated PR https://github.com/bitnami/charts/pull/21318 was merged 3 days ago with some changes that removed the `readOnly: true` attribute in the volume. Could you please check if it resolves the issue? 😁

**Comment by balazs92117** [comment]
Yeah, unforunatelly, I still have the same problem :(

**Comment by balazs92117** [comment]
Also in nginx-ingress-controller-1.9.3 the geoip directory has been changed: https://github.com/kubernetes/ingress-nginx/pull/10470 Maybe this is related somehow.
And btw, as the old legacy geoip has been removed (https://github.com/kubernetes/ingress-nginx/pull/10500/files), the `libgeoip1` is probably not required anymore in https://github.com/bitnami/containers/blob/main/bitnami/nginx-ingress-controller/1/debian-11/Dockerfile#L26C127-L26C136

**Comment by joancafom** [comment]
Hi @balazs92117 

I have checked your issue and was able to reproduce it with my own key! I will work on some changes to bring back the feature, will keep this thread updated with news 😄

**Comment by joancafom** [comment]
Hi @balazs92117 

Just to let you know we have just released new version of our products that are aimed to address this issue. Now, installing the chart with the `maxmindLicenseKey` option will actually download the relevant files:

```
$ helm install nic oci://registry-1.docker.io/bitnamicharts/nginx-ingress-controller --set maxmindLicenseKey="XXXXX"
$ k exec -it nic-nginx-ingress-controller-d5986c6c5-gw7cq -- ls /etc/ingress-controller/geoip
GeoLite2-ASN.mmdb  GeoLite2-City.mmdb
```

Additionally, a new version of the `bitnami/nginx-ingress-controller` image has been released dropping the no longer `libgeoip1` package see https://github.com/bitnami/containers/commit/1b54f6c4dd745a6e4c311dde5e630f2d691a23fa

I will proceed to close this issue now, but feel free to reach out to us if you have any comments or create a new issue if you feel the need to

Thanks! 🚀

**Comment by balazs92117** [comment]
It works now, thank you!
